GenerateUniqueData/example.create.ps1

```diff
@@ -1,9 +1,9 @@
 #####################################################
```

Comment:
Gaan we een PSv1 voorbeeld zo bewaren? Even met Ramon checken misschien?

Reply:
Met Ramon gecheckt. We bewaren het voorbeeld (hij heeft het al 3x gebruikt, maar passen het niet aan naar v2)